examples/meta/src/evaluation/cross_validation_support_vector_machine.sg

```diff
@@ -8,11 +8,11 @@ Machine svm = machine("LibLinear")
 svm.put("liblinear_solver_type", enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
 
 StratifiedCrossValidationSplitting splitting_strategy(labs, 5)
-AccuracyMeasure evaluation_criterion()
+Evaluation evaluation_criterion = evaluation("AccuracyMeasure")
 CrossValidation cross(svm, feats, labs, splitting_strategy, evaluation_criterion)
 cross.set_num_runs(2)
 
 CrossValidationResult result = CrossValidationResult:obtain_from_generic(cross.evaluate())
```

Comment:
`EvaluationResult result = cross.evaluate()` didnt work?

Reply:
my mistake, just testing the patch and pushing once done.